src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2984,13 +2992,16 @@ public IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predicate, Sym
                 spine.Add(current);
 
                 var container = GetSpineSymbol(spine);
-                foreach (var member in container.GetMembers())
```

Comment:
Not sure why this diff looks so painful, but I just wrapped the `foreach` in an `if` block.